app/src/main/java/com/ably/tracking/demo/publisher/api/RetrofitDeliveryServiceApiSource.kt

```diff
@@ -0,0 +1,15 @@
+package com.ably.tracking.demo.publisher.api
+
+class RetrofitDeliveryServiceApiSource(private val deliveryServiceApi: DeliveryServiceApi) :
```

Comment:
I don't think that this class should contain `Retrofit` in its name (even though it's using it) :thinking: For me, a more meaningful name would be one that informs me that this class returns data from some remote async backend, so perhaps `Online` or `Remote` or `Backend` instead would be a better choice? WDYT? :wink:

Reply:
I think ApiSource already suggests interactions with remote backend 🤔 We can always go back to `Default` 😅

Reply:
Hmm this makes me think that maybe we should drop the `Api` from the source as it already defines the type of source :thinking: With the current name we can't have for example a `DatabaseDeliveryServiceApiSource` right? If we change the interface name to `DeliveryServiceSource` (or `DeliveryServiceDataSource`) then we can have `ApiDeliveryServiceSource` and `LocalDeliveryServiceSource` and others, WDYT? :wink:

Reply:
1c1ee251c571dd9da02620ef1fd4e898eac75da0